benchmark/preconditioner/preconditioner.cpp

```diff
@@ -67,6 +66,8 @@ DEFINE_double(accuracy, 1e-1,
               "This value is used as the accuracy flag of the adaptive Jacobi "
               "preconditioner.");
 
+DECLARE_string(formats);
```

Comment:
missing default value and description ?

Reply:
No, it is defined in `utils/general.hpp`, so I can not redefine it
Same as formats.
Only change the default value but keep the same descriptions.

Reply:
Ah, okay.